module/VuFind/src/VuFind/ILS/Driver/DAIA.php

```diff
@@ -182,6 +194,16 @@ public function init()
         } else {
             $this->debug('No ContentTypes for response defined. Accepting any.');
         }
+        if (isset($this->config['DAIA']['daiaServices'])) {
+            $this->daiaServices = explode(
+                ':',
+                $this->config['DAIA']['daiaServices']
+            );
+        } else {
+            $this->debug(
+                'Accepting loan, presentation, remote and openaccess as services.'
```

Comment:
Improved debugging message: "No services configured - accepting loan, presentation, remote and openaccess as services by default."